backend/fms_core/tests/invalid_templates/Project_link_samples_invalid_project_and_sample.csv

```diff
@@ -0,0 +1,108 @@
+﻿Project link samples Template,,,,,
+,,,,,
+Instructions:,,,,,
+,Each line define the addition or removal of a sample instance from a Project. Each sample instance will ,,,,
+,inherit from its parent’s association when extracted or transfered. A sample instance can be associated,,,,
+,to more than one project.,,,,
+,,,,,
+#,Action,Project Name,Sample Name,Sample Container Barcode,Sample Container Coord
+1,Add,TestProject1,InvalidSample,tube001,
+2,Add,InvalidProject,sample2,tube002,
+3,,,,,
+4,,,,,
+5,,,,,
+6,,,,,
+7,,,,,
+8,,,,,

```

Comment:
You can cut all those extra lines. They are not really useful. You can leave one so it tests the case where there is empty lines in the template but the rest are not of any use.